app/forms/admin/claims_filter_form.rb

```diff
@@ -25,6 +25,8 @@ def claims
         Claim.includes(:decisions).held.awaiting_decision
       when "failed_bank_validation"
         Claim.includes(:decisions).failed_bank_validation.awaiting_decision
+      when "awaiting_provider_verification"
+        Claim.awaiting_fe_provider_verification
```

Comment:
```suggestion
        Claim.by_policy(Policies::FurtherEducationPayments).awaiting_fe_provider_verification
```
I think this would be clearer.